api/devices/schema.py

```diff
@@ -96,7 +98,13 @@ class Query(graphene.ObjectType):
 
     def resolve_all_devices(self, info):
         query = Devices.get_query(info)
-        return query.order_by(func.lower(DevicesModel.name)).all()
+        location_id = admin_roles.user_location_for_analytics_view()
+        location_query = LocationSchema.get_query(info)
+        result = location_query.filter(LocationModel.state == "active")
```

Comment:
User more specific variable names. Avoid general names like result. Can you query for both state and id in one statement?